addons/web_editor/static/src/components/media_dialog/file_selector.js

```diff
@@ -257,6 +272,16 @@ export class FileSelector extends Component {
             && this.constructor.tagNames.includes(this.props.media.tagName)
             && !this.selectedAttachmentIds.length;
     }
+
+    /**
+     * This is used (debounced) to be called after loading an attachment.
+     * This way, the user can always see the "load more" button.
+     */
+    scrollToLoadMoreButton() {
+        if (this.state.needle || this.state.attachments.length > this.NUMBER_OF_ATTACHMENTS_TO_DISPLAY) {
+            this.loadMoreButtonRef.el.scrollIntoView({ block: 'end', inline: 'nearest', behavior: 'smooth' });
```

Comment:
@qsm-odoo In order for this commit to be safe for stable, I think an extra check on `this.loadMoreButtonRef.el` is needed before calling `scrollIntoView` on it (if the js was updated but not the xml, the ref will be empty).